packages/components/pages/f-account-info/test-utils/component-objects/f-account-info-selectors.js

```diff
@@ -0,0 +1,46 @@
+exports.COMPONENT = '[data-test-id="account-info"]';
+exports.CHANGE_EMAIL_ADDRESS_LINK = '[name="account-info-change-emailAddress-link"]';
+exports.SAVE_CHANGES_BUTTON = '[data-test-id="account-info-save-changes-button"]';
+exports.CHANGE_PASSWORD_BUTTON = '[data-test-id="account-info-change-password-button"]';
+exports.DELETE_ACCOUNT_LINK = '[name="account-info-delete-account-link"]';
+
+
+exports.FIELDS = {
+    firstName: {
+        input: '[data-test-id="formfield-account-info-consumer-firstName-input"]',
+        emptyError: '[data-test-id="consumer-firstName-empty-error"]',
+        invalidError: '[data-test-id="consumer-firstName-invalid-error"]'
+    },
+    lastName: {
+        input: '[data-test-id="formfield-account-info-consumer-lastName-input"]',
+        emptyError: '[data-test-id="consumer-lastName-empty-error"]',
+        invalidError: '[data-test-id="consumer-lastName-invalid-error"]'
+    },
+    emailAddress: {
+        input: '[data-test-id="formfield-account-info-consumer-emailAddress"]'
+    },
+    phoneNumber: {
+        input: '[data-test-id="formfield-account-info-consumer-phoneNumber-input"]',
+        emptyError: '[data-test-id="consumer-phoneNumber-empty-error"]',
+        invalidError: '[data-test-id="consumer-phoneNumber-invalid-error"]'
+    },
+    addressLine1: {
+        input: '[data-test-id="formfield-account-info-consumer-line1-input"]',
+        emptyError: '[data-test-id="consumer-address-line1-empty-error"]'
+    },
+    addressLine2: {
+        input: '[data-test-id="formfield-account-info-consumer-line2-input"]'
+    },
+    addressLine3: {
+        input: '[data-test-id="formfield-account-info-consumer-line3-input"]'
+    },
+    city: {
+        input: '[data-test-id="formfield-account-info-consumer-locality-input"]',
+        emptyError: '[data-test-id="consumer-city-empty-error"]'
+    },
+    postcode: {
+        input: '[data-test-id="formfield-account-info-consumer-postcode-input"]',
+        emptyError: '[data-test-id="consumer-postcode-empty-error"]',
+        invalidError: '[data-test-id="consumer-postcode-invalid-error"]'
+    }
+};
```

Comment:
Ditto for these other fields. It seems to me that we should have 'invalid' validation states for the address fields too.

Maybe not something for this PR, but worth raising with @oliversweb perhaps?

Reply:
spoken to @tommcclean about this, we don't require 'invalid' validation states for the address fields - they're just required fields needed to send to the API. However, we need the postcode to be in the correct format (use regex for this), hence why this has the extra validation 👍